web/template/layout/mlpa-case-summary.gohtml

```diff
@@ -0,0 +1,60 @@
+{{ define "mlpa-case-summary" }}
+    <div class="app-!-caseworker-summary">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-full">
+                <div class="govuk-!-display-block">
+                    <h3 class="govuk-heading-s govuk-!-display-inline govuk-!-padding-right-3">Case summary</h3>
```

Comment:
I think this should be a `h2` so it has the hierarchy [Donor name] -> Case summary -> Warnings
```suggestion
                    <h2 class="govuk-heading-s govuk-!-display-inline govuk-!-padding-right-3">Case summary</h2>
```